client/src/game/reducers/cardsReducer.ts

```diff
@@ -173,6 +173,11 @@ const cardsReducer = (
     }
   }
 
+  if (game.turn.turnNum === 0 && action.type === 'draw') {
```

Comment:
Is there a way to tell we've drawn the last initial card?  If someone else draws a five in their starting hand, I think we'd want to start the game with the deduction that we don't have that 5.

Reply:
Ah, I see there is `isInitialDealFinished` under `deck.ts`